docs/transparent-validation/rte/v1.1.0/serial-port-connection-guide.md

```diff
@@ -0,0 +1,43 @@
+# Serial port connection guide
+
+This document describes how to set up a serial connection between RTE and
+another device.
+
+## Prerequisites
+
+* [Prepared RTE](../v1.1.0/quick-start-guide.md)
+* Two jumpers
+* Choose one of the following cables depending on the connectivity to your
+    platform:
+    - RS232-RJ45 cable
+    - RS232-RS232 cable
+    - 3 wire cables
+
+## Setting serial connection
+
+1. Connect the RTE header (J14 or J18) with the device serial port.
+1. Put the two jumpers on
```

Comment:
The putting jumpers section should correspond with the comment above about the determination of the method of connection.

Also, this point should be moved before the point on the connection to the RTE header.

Reply:
Updated section